contracts/finance/andromeda-validator-staking/Cargo.toml

```
@@ -19,6 +19,7 @@ cosmwasm-std = { workspace = true, features = ["staking"]  }
 cosmwasm-schema = { workspace = true }
 cw-storage-plus = { workspace = true }
 cw2 = { workspace = true }
+serde = { version = "1.0.127", default-features = false, features = ["derive"] }
```

This is included in the workspace.

```suggestion
serde = { workspace = true }
```

---

Updated.